Automattic/wp-calypso
# Scheduled updates: better state handling in query

Closes #90014

## Proposed Changes

* Uses getSites selector, to avoid rerendering any time state changes.

## Testing Instructions

- Test for regressions. http://calypso.localhost:3000/plugins/scheduled-updates should load fine, and expanded view should show site titles.


## Pre-merge Checklist
